composer.json

```
@@ -14,8 +14,8 @@
     "require": {
         "php": ">=7.2",
         "guzzlehttp/guzzle": "^6.2",
-        "illuminate/notifications": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0",
-        "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0"
+        "illuminate/notifications": "5.5.*|5.6.*|5.7.*|5.8.*|^6.0|^7.0",
```

```suggestion
        "illuminate/notifications": "~5.5 || ^6.0 || ^7.0",
```

Single pipe is deprecated + can clean up reqs here :)

---

(and below)

---

(also need to allow testbench 4/5 for laravel 6/7